Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -426,6 +428,27 @@ def fromString(value):
 	def toString(value):
 		return fl2str(value, 14)
 
+class Angle(F2Dot14):
+	factor = 1.0/(1<<14) * 180  # 0.010986328125
+	@staticmethod
+	def wrapAround(angle):
+		# Upon compile/dump, normalize angles that span beyond a full circle.
+		# We want to keep the orientation after modulo: e.g. -415 => -55, not 305
+		return angle % math.copysign(360, angle)
+	def read(self, reader, font, tableDict):
+		return super().read(reader, font, tableDict) * 180
+	def write(self, writer, font, tableDict, value, repeatIndex=None):
+		value = self.wrapAround(value)
+		super().write(writer, font, tableDict, value / 180, repeatIndex=repeatIndex)
+	@classmethod
+	def fromString(cls, value):
+		# quantize to nearest multiples of minimum fixed-precision angle
+		return otRound(float(value) / cls.factor) * cls.factor
```

Comment:
I would expect that you call something ala `fixedToFloatToFixed` here as well.

Reply:
you meant `floatToFixedToFloat` or `strToFixedToFloat`. The code above is basically doing that, but on a value which represents degrees [-360..360), not fixed-point values [-2.0..2.0) with exact `precisionBits` (the second parameter to the above mentioned functions).

in other words, my code above is doing this:

```python
fixedToFloat(floatToFixed(float(value) / 180, 14), 14) * 180
```

I can change to this if it's preferable